sdk/src/main/java/com/kin/ecosystem/Kin.java

```diff
@@ -215,6 +219,7 @@ public void onResponse(AuthToken authToken) {
 				AuthRepository.getInstance().updateWalletAddress(publicAddress, new KinCallback<Boolean>() {
 					@Override
 					public void onResponse(Boolean response) {
+						eventLogger.send(UserLoginSucceeded.create());
```

Comment:
Need to talk with BI - we planned to only send login  if user currently logout or login as a different user - "when a user login with a different device /user_id , and this device is currently logout"

Reply:
done already.